util/load_pokemons.go

```diff
@@ -0,0 +1,67 @@
+package util
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/tamnguyenwizeline/ondemand-go-bootcamp/model"
+)
+
+type PokemonResponse struct {
+	model.Pokemon
+	URL string
+}
+
+func LoadPokemonData(filePath string) error {
+	url := "https://pokeapi.co/api/v2/pokemon?limit=100000&offset=0"
```

Comment:
This could be passed through parameter in order to read it from environment configs